incubator/cert-manager-setup/ci/test-values.yaml

```diff
@@ -0,0 +1,10 @@
+clusterIssuers:
```

Comment:
AFAICT this does test the DNS solver configuration at all.  Can this be expanded to do that?

Reply:
Added dns solver